data/mods/Dark-Skies-Above/mapgen/map_extras/patrol_test.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_dsa_patrol_test",
```

Comment:
This seems unrelated to fake vehicle parts stuff?

Reply:
Thanks for reminding me, this might be a merge error or something, I need to look into reverting it.